stock_picking_report_custom_description/models/sale_order_line.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    def write(self, vals):
+        res = super().write(vals)
+        if vals.get("name"):
+            self.env["stock.move"].search(
+                [("sale_line_id", "in", self.ids)]
+            ).name = vals["name"]
```

Comment:
Theoretically, this can raise a singleton error. Additionally, why not use the `move_ids` field instead of performing a search?
```suggestion
            self.env["stock.move"].search([("sale_line_id", "in", self.ids)]).name = vals[
                "name"
            ]
```

Reply:
Sorry, I see that ids was updated while I was writing my review.